serial/serial_resources/component_instance_link.py

```diff
@@ -0,0 +1,5 @@
+import serial
+
+class ComponentInstanceLink:
```

Comment:
Very basic class for storing ComponentInstanceLink Data. Would be great to not have to implement all of it.

Reply:
Agreed! I think it would be reasonable to leave the reading of links out of the python library entirely for this release. We can try to tackle that beast later